dashboard/app/models/pd/enrollment.rb

```diff
@@ -259,22 +266,22 @@ def school_district_name
   end
 
   def update_scholarship_status(scholarship_status)
-    if workshop.local_summer?
-      Pd::ScholarshipInfo.update_or_create(user, workshop.summer_workshop_school_year, workshop.course_key, scholarship_status)
+    if workshop.local_summer? || workshop.csf?
+      Pd::ScholarshipInfo.update_or_create(user, workshop.school_year, workshop.course_key, scholarship_status)
     end
   end
 
   def scholarship_status
-    if workshop.local_summer?
-      Pd::ScholarshipInfo.find_by(user: user, application_year: workshop.summer_workshop_school_year, course: workshop.course_key)&.scholarship_status
+    if workshop.local_summer? || workshop.csf?
+      Pd::ScholarshipInfo.find_by(user: user, application_year: workshop.school_year, course: workshop.course_key)&.scholarship_status
     end
   end
 
   # Returns true if this enrollment is for a novice or apprentice facilitator (accepted this year)
   # attending a local summer workshop as a participant to observe the facilitation techniques
   def newly_accepted_facilitator?
     workshop.local_summer? &&
-      workshop.summer_workshop_school_year == APPLICATION_CURRENT_YEAR &&
+      workshop.school_year == APPLICATION_CURRENT_YEAR &&
       FACILITATOR_APPLICATION_CLASS.where(user_id: user_id).first&.status == 'accepted'
```

Comment:
nit: should this next line be indented?

Reply:
I'm not sure! should it? it's the same level as 284, both of which are continuations of 283

Reply:
Oops, of course.